src/worker.js

```diff
@@ -22,65 +58,14 @@ async function worker(options) {
     return result;
   }
 
-  /**
-   * @param {WorkerResult} item
-   * @param {undefined | string | FilenameFn} filename
-   * @returns {WorkerResult}
-   */
-  const normalizeProcessedResult = (item, filename) => {
-    if (!item.info) {
-      item.info = {};
-    }
-
-    if (!item.filename) {
-      item.filename = options.filename;
-    }
-
-    if (!item.errors) {
-      item.errors = [];
-    }
-
-    if (!item.warnings) {
-      item.warnings = [];
-    }
-
-    if (options.severityError === "off") {
-      item.warnings = [];
-      item.errors = [];
-    } else if (options.severityError === "warning") {
-      item.warnings = [...item.warnings, ...item.errors];
-      item.errors = [];
-    }
-
-    if (
-      typeof filename !== "undefined" &&
-      typeof options.generateFilename !== "undefined" &&
-      !item.info.original
-    ) {
-      item.filename = options.generateFilename(filename, {
-        filename: item.filename,
-      });
-    }
-
-    if (item.info.original) {
-      delete item.info.original;
-    }
-
-    return item;
-  };
-
   const transformers = Array.isArray(options.transformer)
     ? options.transformer
     : [options.transformer];
 
-  for (let i = 0; i <= transformers.length - 1; i++) {
+  for await (const transformer of transformers) {
```

Comment:
Do we need `await` here?

Reply:
yes, because we have a wait statement inside loop
https://github.com/webpack-contrib/image-minimizer-webpack-plugin/blob/0dbc63a18ebffd75e67d77e786c0d41f15fb8910/src/worker.js#L77

Reply:
`transformers` is not async iterable object, it is just array

Reply:
Yes, shure. I just tried remove `eslint-disable-next-line no-await-in-loop` line. It's my fall, I'll fix it